packages/stark-ui/src/modules/pagination/components/pagination.component.ts

```diff
@@ -130,15 +130,15 @@ export class StarkPaginationComponent extends MatPaginator implements OnInit, On
 	}
 
 	/**
-	 * Component lifecycle hook
+	 * CodefaultItemsPerPageIsPresentComponent lifecycle hook
 	 */
 	public ngOnChanges(changesObj: SimpleChanges): void {
 		if (changesObj["paginationConfig"]) {
 			// Set local variable to prevent shadow changes
 			const paginationConfigOriginalChange: StarkPaginationConfig = { ...this.paginationConfig };
 			this.paginationConfig = this.normalizePaginationConfig(this.paginationConfig);
 			this.logger.debug(componentName + ": paginationConfig changed...", this.paginationConfig);
-
+			this.logger.warn(componentName + ": No configuration defined in pagination component. TotalItems set to 0 by default");
```

Comment:
Actually, after rereading the code, I see that this line can be removed. It's a copy of line 168.
That means this message will be displayed twice.

Could you please remove this ? 😊